src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!--<h1>HTML Form</h1>-->
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="form_block">
+        <legend>Personal information </legend>
+        <div class="form_field">
+          <label for="surname">Surname:
+            <input 
+            type="text" 
+            name="Surname" 
+            required 
+            id="surname" 
+            autocomplete="off">
```

Comment:
```suggestion
            <input 
              type="text" 
              name="Surname" 
              required 
              id="surname" 
              autocomplete="off"
            >
```
Зверни увагу на відступи і форматування. Почитати можна тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation